homeassistant/components/metoffice/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
You could make the `homeassistant.components.metoffice.async_setup_entry` patch a shared one, which saves duplication